SConstruct

```diff
@@ -112,6 +112,25 @@ def check_openssl():
             raise Exception("Didn't find any '%s' line in '$ %s'" %
                             (CHECK_LINE, CHECK_COMMAND))
 
+def setImplicitCache():
+    '''Use implicit_cache on some targets to improve build times
+
+    The command line option `--implicit-deps-changed` overrides implicit_cache
```

Comment:
This is a sort of mystery comment.  ;-)  

I read this and ask, er, does that I mean I _am_ supposed to set that flag, or not?  Is this prescriptive or descriptive?

Honestly, I'd start with describing the net result of this function in one sentence - then the rest.